src/common.jl

```diff
@@ -10,25 +10,25 @@
 # covariant type notation, i.e. AbstractVector{<:Real}
 #
 
-@compat RealArray{T<:Real,N} = AbstractArray{T,N}
-@compat RealVector{T<:Real} = AbstractArray{T,1}
-@compat RealMatrix{T<:Real} = AbstractArray{T,2}
+const RealArray{T<:Real,N} = AbstractArray{T,N}
```

Comment:
Yeah, I just personally prefer it since it adds a nice visual distinction from randomly assigning some variable in the middle of package code. 😛